.github/ansible/deploy.yaml

```diff
@@ -59,7 +59,7 @@
         creates: "/storage/pageserver/data/tenants"
       environment:
         NEON_REPO_DIR: "/storage/pageserver/data"
-        LD_LIBRARY_PATH: "/usr/local/v14/lib"
+        LD_LIBRARY_PATH: "/usr/local/v15/lib"
```

Comment:
I wonder why we need LD_LIBRARY_PATH at all? Is it for `psql` and other such client tools?

Reply:
Yep, I was wondering about it as well. @cicdteam may know the reason